docs/components/empty-state.md

```diff
@@ -281,8 +390,99 @@ Slot for providing your custom action button.
 </KEmptyState>
 ```
 
-## Events
+### feature-icon
```

Comment:
issue: it's a bit unintuitive to slot icons via index (a `number`) - is it possible to require each `feature` have a unique `key` instead, and then the `key` is used in the slot name?